test/OpenSpaceToolkit/Astrodynamics/Trajectory/Propagator.test.cpp

```diff
@@ -1914,6 +1914,94 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_Propagator, PropAc
     }
 }
 
+TEST_F(
+    OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_Propagator, PropAccuracy_Drag_Constant_NRLMSISE_Orekit_450km
+)
+{
+    {
+        // Current state and instant setup
+        const Instant startInstant = Instant::DateTime(DateTime::Parse("2023-01-01 00:00:00.000"), Scale::UTC);
+
+        // Reference data setup
+        const Table referenceData = Table::Load(
+            File::Path(Path::Parse("/app/test/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Propagated/"
+                                   "Orekit_Drag_base_plus_drag_test.csv")),
+            Table::Format::CSV,
+            true
+        );
+
+        Array<Instant> instantArray = Array<Instant>::Empty();
+        Array<Vector3d> referencePositionArrayGCRF = Array<Vector3d>::Empty();
+        Array<Vector3d> referenceVelocityArrayGCRF = Array<Vector3d>::Empty();
+
+        for (const auto& referenceRow : referenceData)
+        {
+            instantArray.add(Instant::DateTime(
+                DateTime::Parse(referenceRow[0].accessString(), DateTime::Format::ISO8601), Scale::UTC
+            ));
+
+            referencePositionArrayGCRF.add(
+                Vector3d(referenceRow[1].accessReal(), referenceRow[2].accessReal(), referenceRow[3].accessReal())
+            );
+            referenceVelocityArrayGCRF.add(
+                Vector3d(referenceRow[4].accessReal(), referenceRow[5].accessReal(), referenceRow[6].accessReal())
+            );
+        }
+
+        // Setup dynamics
+        const Earth earth = Earth::FromModels(
+            std::make_shared<EarthGravitationalModel>(EarthGravitationalModel::Type::Spherical),
+            std::make_shared<EarthMagneticModel>(EarthMagneticModel::Type::Undefined),
+            std::make_shared<EarthAtmosphericModel>(EarthAtmosphericModel::Type::NRLMSISE00)
+        );
+        const Shared<Celestial> earthSPtr = std::make_shared<Celestial>(earth);
+        const Array<Shared<Dynamics>> dynamics = {
+            std::make_shared<PositionDerivative>(),
+            std::make_shared<CentralBodyGravity>(earthSPtr),
+            std::make_shared<AtmosphericDrag>(earthSPtr, satelliteSystem_),
+        };
+
+        // Setup initial conditions
+        const State state = {
+            startInstant,
+            Position::Meters({referencePositionArrayGCRF[0]}, gcrfSPtr_),
+            Velocity::MetersPerSecond({referenceVelocityArrayGCRF[0]}, gcrfSPtr_),
+        };
```

Comment:
```suggestion
        const State state = {
            startInstant,
            Position::Meters(referencePositionArrayGCRF[0], gcrfSPtr_),
            Velocity::MetersPerSecond(referenceVelocityArrayGCRF[0], gcrfSPtr_),
        };
```
Are the {} braces necessary?